src/js/main.js

```diff
@@ -233,17 +258,21 @@ if (document.body.dataset.page === "run") {
     window.location.href = "/";
   }
 
-  settingsStore.getAndSubscribe(({ livebookUrl }) => {
-    const livebookImportUrl = getLivebookImportUrl(livebookUrl, notebookUrl);
+  settingsStore.getAndSubscribe(({ livebookUrl, useLivebookDesktop }) => {
+    const livebookImportUrl = getLivebookImportUrl((useLivebookDesktop ? "livebook://" : livebookUrl), notebookUrl);
```

Comment:
this is a naïve placeholder and not yet correct

Reply:
You need to install the desktop app for it. An alpha version for mac is available here: https://github.com/livebook-dev/livebook/releases/tag/v0.5.1

Reply:
The idea is that we will do something like this for the desktop case: `notebookUrl.replace(/^https?:/, "livebook:")`. you don't need to call importUrl.

Reply:
Implemented 👍